zeppelin-server/src/test/java/org/apache/zeppelin/integration/ParagraphActionsIT.java

```diff
@@ -55,6 +56,33 @@ public void tearDown() {
 
     driver.quit();
   }
+  @Test
+  public void testCreateNewButton() throws InterruptedException {
+    if (!endToEndTestEnabled()) {
+      return;
+    }
+    try {
+      createNewNote();
+
+      waitForParagraph(1, "READY");
+      Integer nosOfParas =    driver.findElements(By.xpath("//div[@ng-controller=\"ParagraphCtrl\"]")).size();
+      driver.findElement(By.xpath(getParagraphXPath(1) + "//span[@class='icon-settings']")).click();
+      driver.findElement(By.xpath(getParagraphXPath(1) + "//ul/li/a[@ng-click='insertNew()']")).click();
+      waitForParagraph(2, "READY");
+      Integer newNosOfParas =  driver.findElements(By.xpath("//div[@ng-controller=\"ParagraphCtrl\"]")).size();
+      collector.checkThat("The number of new paragraph and old paragraph",
+              (nosOfParas+1==newNosOfParas),
+              CoreMatchers.equalTo(true));
+
```

Comment:
collector.checkThat("The number of new paragraph and old paragraph",
              nosOfParas + 1,
              CoreMatchers.equalTo(newNosOfParas));